website/docs/addon_applications_admin.md

```diff
@@ -1,63 +1,294 @@
+::: -->
 
-![settings_applications](assets/settings/applications_01.png)
+![settings_applications](assets/applications/admin_applications_layers.png)
 
-### Environments
+The Addon features two primary settings categories: Definitions and Filters:
 
-Please keep in mind that the environments are not additive by default, so if you are extending variables like
-`PYTHONPATH`, or `PATH` make sure that you add themselves to the end of the list.
+1. **Definitions:** available only in studio settings, allows to define all possible applications and tools that can be used across all projects.
+2. **Filters:** defines when certain applications and tools are used on project level. This filters whether a certain application or tool should be enabled in a certain launch context, like a specific task type or project-wide.
 
-For instance:
+## Applications Addon Settings
+### Show only available applications
+![](assets/applications/show_only_available_apps.png)
 
-```json
-{
-    "PYTHONPATH": [
-        "my/path/to/python/scripts",
-        "{PYTHONPATH}"
-    ]
-}
-```
+When enabled, AYON Launcher will display only those applications for which executable paths exist on the current machine.
+This acts as an extra filter to the Applications filters, ensuring that inaccessible applications are excluded from view.
 
-### Adding versions
+### Applications Definitions
 
-It is possible to add new version for any supported application. There are two ways of doing it.
+In this section you can manage what Applications are available to your studio, locations of their executables, and their additional environments. 
+In AYON context, each application that is integrated is also called a `Host` and these two terms will be used interchangeably in the documentation.
 
-1. **Add new executable** to an existing application version. This is a good way if you have multiple fully compatible versions of your DCC across the studio. Nuke is a typical example where multiple artists might have different `v#` releases of the same minor Nuke release. For example `12.2v3` and `12.3v6`. When you add both to `12.2` Nuke executables they will be treated the same in AYON and the system will automatically pick the first that it finds on an artist machine when launching. Their order is also the order of their priority when choosing which version to run if multiple are present.
-![settings_applications](assets/settings/settings_addapplication.gif)
+#### Built-in Applications
+Each application definition has an application group e.g. Maya, Houdini, Nuke and etc.
+
+![settings_applications](assets/applications/application_grorup.png)
+
+Each Application is made of two levels.
+1. **Application group:**
+   1. **Host name:** Pipeline integration name supplied by an AYON addon.
```

Comment:
Since, I've removed `What is host name used for?` section. https://github.com/ynput/ayon-documentation/pull/220/commits/6bd703b62de767d53e9b4628cf92fa40e8be05e7

should we rephrase this line ? 
Because, it bothers me reading through the documentation without a clue what should this variable used for and what value should I use for it.

Reply:
I suppose just:
```suggestion
   1. **Host name:** Pipeline integration name supplied by an AYON addon, these are usually constants related to a host integration like "maya", "houdini", "nuke" and so forth. For now they are best left at the default values.
```